### ISSUE TITLE
Display List Item: On accessing the list item as highlighted, UI type is conveyed as ‘Link’.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3835]
Description: On accessing the list item as highlighted, UI type is conveyed as ‘Link’.

Expected: UI type should be conveyed according to the role assigned given for the UI

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98774934-082b5180-2412-11eb-99a6-13c5cfcc1641.png)

**Comment by sKudum** [comment]
Updated voice message to "list item with link "
